HW_3/task_2.py

```diff
@@ -0,0 +1,26 @@
+"""
+Задайте список целых чисел. Верните список с произведениями его парных элементов.
+Парой считаются первый и последний элемент, второй и предпоследний и т.д.
+Если элементов нечетное количество – центральный элемент умножается сам на себя.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[2, 3, 4, 5, 6]
+[12, 15, 16]
+
+[2, 3, 5, 6]
+[12, 15]
+"""
+from random import randint
+# from random import randint # randint возвращает случайное целочисленное значение
+
+number = int(input('Введите число '))
+first_lst = [randint(0, 10) for i in range(number)]
+print(first_lst)
+
+second_lst = [] # новый лист, в котором будет считаться результат
+for i in range(int((number) // 2) + 1): # размерность  числа / на цело на 2 и до крайнего значения т.е. +1
```

Comment:
Артем, сожалею, но это неверно. Во-первых, при целочисленном делении не нужно переводить в тип `int`. Во-вторых, при размерности списка больше 5, у  вас индекс середины будет высчитываться неверно. И, как результат, появляется лишний элемент в результирующем списке.
![image](https://user-images.githubusercontent.com/49741194/205813148-5247d2a1-a17e-4e5f-b96c-8de8aa087c3b.png)
Нужно делать так
```suggestion
for i in range((inumber + 1) // 2): # размерность  числа / на цело на 2 и до крайнего значения т.е. +1
```

Reply:
Исправил